src/style.css

```diff
@@ -2,2 +1,70 @@
+.first {
+  margin: 0;
+}
```

Comment:
Avoid using the universal selector '*' for setting styles. It impacts browser performance as it applies the styles to all elements in the DOM. It's better to apply styles to specific classes that need them.